### DISCUSSION TITLE
Make changes to the app

### DISCUSSION THREAD

**Discussion by scoopy21** [Imperial-lord/mapbox-flutter#11] [Q&A]
How can I amend the app to be useful for blind people? I want to be able to add speech to text on the destination text field. I have been able to do the speech-to-text functionality separately but I don't know where to add it. 
I also want the source controller to always be the current location of the user.

Could you please guide me? Would be really grateful

**Comment by Imperial-lord** [comment]
This will need some discussion - can you tell me about the purpose? Where all do you need the speech to text functionality? I think with Android accessibility options blind people can navigate through the app, and voice instructions will be established by default with the maps.
